src/utils/createAdaptorShape.js

```diff
@@ -0,0 +1,7 @@
+export default function createAdaptorShape(PropTypes) {
+  return PropTypes.shape({
+    resolve: PropTypes.func.isRequired,
+    subscribe: PropTypes.func.isRequired,
+    shouldComponentUpdate: PropTypes.func.isRequired,
```

Comment:
It would be helpful, in lieu of API docs to put some JSDoc here describing the function signature.